src/ipa/rpi/controller/rpi/sync.cpp

```diff
@@ -119,100 +139,175 @@ void Sync::initialise()
 	socket_ = -1;
 }
 
+/* No idea what this does*/
 void Sync::switchMode([[maybe_unused]] CameraMode const &cameraMode, [[maybe_unused]] Metadata *metadata)
 {
 	syncReady_ = false;
 	frameCount_ = 0;
 	readyCountdown_ = 0;
 }
 
+
+/* Most important part, algorithm*/
```

Comment:
We could probably supply a more useful comment, maybe one that explains the algorithm a bit? I don't think it has to say too much, e.g.
```
/*
 * Camera sync algorithm.
 *     Server - there is a single server that sends framerate timing information over the network to any
 *         clients that are listening. It also signals when it will send a "everything is synchronised, now go"
 *         message back to the algorithm.
 *     Client - there may be many clients, either on the same Pi or different ones. They match their
 *         framerates to the server, and indicate when to "go" at the same instant as the server. 
 */
```
 It's also convention to have only one blank line between functions.